pyhoma/enums.py

```diff
@@ -73,6 +73,7 @@ class DataType(Enum):
     BLOB = 4
     DATE = 5
     BOOLEAN = 6
+    PASSWORD = 9
```

Comment:
I don’t get why Somfy expose the password.

Reply:
This is the case of GenericCamera's linked to the Somfy system. I assume they need the user + pass to connect to the stream via RTSP...